compiler/rustc_session/src/options.rs

```diff
@@ -532,6 +534,18 @@ macro_rules! options {
             true
         }
 
+        fn parse_debuginfo(slot: &mut DebugInfo, v: Option<&str>) -> bool {
+            match v {
+                Some("0") | Some("none") => *slot = DebugInfo::None,
+                Some("line-directives-only") => *slot = DebugInfo::LineDirectivesOnly,
+                Some("line-tables-only") => *slot = DebugInfo::LineTablesOnly,
+                Some("1") | Some("limited") => *slot = DebugInfo::Limited,
```

Comment:
Is there value in adding additional aliases to `-Cdebuginfo=1` if it generates full debug-info (same as 2/full) anyway?

Reply:
Yes, because it doesn't generate the same debug information as 2/full, see #69080 for details, which makes 1/limited the same as clang's `-debug-info-kind=limited` and prevents #60020 from reoccurring.
[clang implementation](https://github.com/llvm/llvm-project/blob/a92dbadffe4ff81cbae7303b0f3e723e26fa77b1/clang/lib/CodeGen/CGDebugInfo.cpp#L595)